src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateExcludeDateRepository.kt

```diff
@@ -1,8 +1,19 @@
 package uk.gov.justice.digital.hmpps.visitscheduler.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Modifying
+import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
 import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.SessionTemplateExcludeDate
+import java.time.LocalDate
 
 @Repository
-interface SessionTemplateExcludeDateRepository : JpaRepository<SessionTemplateExcludeDate, Long>
+interface SessionTemplateExcludeDateRepository : JpaRepository<SessionTemplateExcludeDate, Long> {
```

Comment:
Question I've just thought of related to this new table. Do we need a constraint on the Table to make it that there is a restriction on this combination (session_template_id + exclude_date). So that you can only have 1 instance of each? To stop people potentially creating duplicate entries.

Reply:
makes sense, I will add a DB constraint.